libs/runtime/ebpf_object.c

```diff
@@ -260,10 +183,35 @@ ebpf_object_initialize(
     object->type = object_type;
     object->free_function = free_function;
     object->get_program_type = get_program_type_function;
+    object->id = ebpf_interlocked_increment_int32((volatile int32_t*)&_ebpf_next_id);
```

Comment:
The [docs](https://learn.microsoft.com/en-us/windows/win32/api/winnt/nf-winnt-interlockedincrement) indicate that the argument needs to be 32-bit aligned.  We might want to err on the side of caution and explicitly declare `_ebpf_next_id` as 32-bit aligned?

Reply:
Integer types are always naturally aligned. i.e. a 32bit value is always 32bit aligned.